core/primitives/src/telemetry.rs

```diff
@@ -31,6 +31,14 @@ pub struct TelemetryChainInfo {
     pub latest_block_hash: String,
```

Comment:
Unrelated to this PR but this should just be CryptoHash.  CryptoHash serialises as bas58 in JSON.

Reply:
Thanks, done.